### PR TITLE
Adds Profiles to civicrm_admin_ui

### PR DESCRIPTION
**Pull request by AlainBenbassat**
Overview
----------------------------------------
Adds Profiles to civicrm_admin_ui.

Before
----------------------------------------
Quickform admin screens for profiles and profile fields.

After
----------------------------------------
Search Kit + Afform to manage profiles and profile fields.

Comments
----------------------------------------
- I could not display the "Used for" in the profile admin screen because there are no pseudoconstants for that.
- I could not add the help text behind the question mark in the profile admin screen.

### CODE REVIEW

File: ext/civicrm_admin_ui/ang/afsearchProfileFields.aff.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "is_token": false,
+    "contact_summary": null,
+    "icon": "fa-list-alt",
+    "server_route": "civicrm/admin/uf/group/field",
+    "permission": "access CiviCRM",
+    "redirect": null,
+    "create_submission": false,
+    "navigation": {
+        "parent": "Customize Data and Screens",
+        "label": "Profile Fields",
```
Comment (Damilare1) [inline ext/civicrm_admin_ui/ang/afsearchProfileFields.aff.json:19]: We need to add internalisation to the label and text strings.

File: ext/civicrm_admin_ui/managed/SavedSearch_Administer_Profiles.mgd.php [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  [
+    'name' => 'SavedSearch_User_defined_Profiles',
+    'entity' => 'SavedSearch',
+    'cleanup' => 'unused',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'User_defined_Profiles',
+        'label' => 'Administer Profiles',
```
Comment (Damilare1) [inline ext/civicrm_admin_ui/managed/SavedSearch_Administer_Profiles.mgd.php:14]: We need to add internalisation to the label and text strings.

**Review by Damilare1 — CHANGES_REQUESTED** [review]
Hi @AlainBenbassat, thanks for working on this. I dropped comments on some places I believe could do with some internationalisation using the CRM_CivicrmAdminUi_ExtensionUtil class.

**Comment by colemanw** [comment]
I've pushed up some additional changes to this, and I think it's close enough to parity with the original form to be merged at this point.

**Comment by AlainBenbassat** [comment]
Thank you @colemanw for finishing my work.

**Comment by colemanw** [comment]
No problem @AlainBenbassat  - perhaps you could take a moment to leave a review on #24759 :)

**Comment by larssandergreen** [comment]
I believe there is a bit of an issue with how the redirection on adding a profile and the breadcrumbs work in this change, [issue here (#4254).](https://lab.civicrm.org/dev/core/-/issues/4254)

**Pull request merged**